packages/flutter_tools/lib/src/commands/ios.dart

```diff
@@ -59,6 +59,11 @@ class IOSCommand extends FlutterCommand {
     tempFile.writeAsBytesSync(archiveBytes);
 
     try {
+      // Remove the old generated project if one is present
+      runCheckedSync(['/bin/rm', '-rf', directory]);
```

Comment:
We should probably prefer `dart:io` functions where they exist, over shelling out. So, `new Directory(directory).deleteSync(recurse: true)`.